structures/freighter.py

```diff
@@ -26,16 +30,34 @@ def pop(self) -> str:
         returns an empty string if the containers stack is empty, instead 
         of an error
         """
-        return ''
+        # TODO implement pop()
+        # FILO = First in last out
+        # FIFO = First in First Out 
+        if self.containers == []:
+            return ''
+        # returns an open strin if the container list is equal to an empty list 
```

Comment:
-1 style, comments should go directly above or next to the code they describe (not below)